### PR TITLE
Replace string concatenations with template literals in clamd.js

### PR DESCRIPTION
**Pull request by pvsousalima**
Fixes #2129 

Changes proposed in this pull request:
- Replace string concatenations with template literals in clamd.js

Checklist:
- [ ] docs updated
- [ ] tests updated
- [ ] [Changes](https://github.com/haraka/Haraka/blob/master/Changes.md) updated

### CODE REVIEW

File: plugins/clamd.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ exports.load_excludes = function () {
                         new_skip_list_exclude.push(re);
                     }
                     catch (e) {
-                        plugin.logerror(e.message + ' (entry: ' + list[i] + ')');
+                        plugin.logerror(e.message + ` (entry: ${list[i]})`);
```
Comment (msimerson) [inline plugins/clamd.js]: any reason to leave `e.message` out of the template literal?

Comment (pvsousalima) [reply plugins/clamd.js]: Will put it back inside.

File: plugins/clamd.js [hunk 2]
```diff
@@ -299,11 +299,11 @@ exports.hook_data_post = function (next, connection) {
             // The current host returned an unknown result.  If other hosts are available,
             // then try those before returning a DENYSOFT.
             if (hosts.length) {
-                connection.logwarn(plugin, 'unknown result: "' + result + '" from host ' + host);
+                connection.logwarn(plugin, `unknown result: "${result}" from host ' + ${host}`);
```
Comment (msimerson) [inline plugins/clamd.js]: You've got a stray ' in there (and below)

Comment (pvsousalima) [reply plugins/clamd.js]: Just pushed the version with the stray '

File: plugins/clamd.js [hunk 3]
```diff
@@ -299,11 +299,11 @@ exports.hook_data_post = function (next, connection) {
             // The current host returned an unknown result.  If other hosts are available,
             // then try those before returning a DENYSOFT.
             if (hosts.length) {
-                connection.logwarn(plugin, 'unknown result: "' + result + '" from host ' + host);
+                connection.logwarn(plugin, `unknown result: '${result}' from host ' + ${host}`);
                 socket.destroy();
                 return try_next_host();
             }
-            txn.results.add(plugin, { err: 'unknown result: "' + result + '" from host ' + host });
+            txn.results.add(plugin, { err: `unknown result: '${result}' from host ' + ${host}`});
```
Comment (msimerson) [inline plugins/clamd.js]: heh heh, sorry, still not quite. Right after the word 'host' is the extra chars: `' +`

Comment (pvsousalima) [reply plugins/clamd.js]: hehehehe sorry

Comment (pvsousalima) [reply plugins/clamd.js]: Pushed again

**Comment by pvsousalima** [comment]
:+1:

**Pull request merged**